### ISSUE TITLE
LocalNotifications schedule interval to start after a future date

### ISSUE THREAD

**Issue by mpkasp** [ionic-team/capacitor-plugins#1114]
## Feature Request
It would be nice if one could schedule a notification at an interval that starts _after_ a particular date, similar to the `after` property in the Cordova Local Notifications plugin.

### Plugin
LocalNotifications

### Description
For example:

```typescript
 notifications: [{
   // ....
   schedule: {
     on: {hour, minute}
    }
   // ....
 }]
```

This schedules a notification that triggers every day at `hour` and `minute` starting the next time we hit `hour` and `minute`.

What I'm looking for something that triggers every day at `hour` and `minute` _after_ a particular date, so for example:

```typescript
 notifications: [{
   // ....
   schedule: {
     on: {hour, minute},
     after: ... // some date in the future
    }
   // ....
 }]
```

### Platform(s)
Minimum: iOS, Android

### Preferred Solution
See description

### Alternatives
Open to any implementation that accomplishes the ability to start an interval notification _after_ a particular future date.

### Additional Context
https://github.com/katzer/cordova-plugin-local-notifications

**Comment by mpkasp** [comment]
Here is the Android implementation https://github.com/ionic-team/capacitor-plugins/commit/329b32eebbbb9aa8bca98b18f272d6bc707529e9 ; if all looks OK I'm happy to support with iOS as well

**Comment by mpkasp** [comment]
I don't have a need for iOS right now & did not get a response so I did not build that implementation

**Comment by eric-horodyski** [comment]
This thread has been inactive for some time, and we’re cleaning up older items to keep the backlog focused and manageable.

Details on the effort:
[https://ionic.io/blog/keeping-capacitors-backlog-healthy](https://ionic.io/blog/keeping-capacitors-backlog-healthy)

If this is still relevant today, feel free to open a new issue with current information and we’ll review it.
